cdap-proto/src/main/java/co/cask/cdap/proto/metadata/MetadataSearchResultRecord.java

```diff
@@ -44,26 +38,22 @@ public boolean equals(Object o) {
     if (this == o) {
       return true;
     }
-    if (o == null || getClass() != o.getClass()) {
+    if (!(o instanceof MetadataSearchResultRecord)) {
       return false;
     }
-
     MetadataSearchResultRecord that = (MetadataSearchResultRecord) o;
-
-    return Objects.equals(this.id, that.id) &&
-      Objects.equals(this.type, that.type);
+    return Objects.equals(entityId, that.entityId);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(id, type);
+    return Objects.hash(entityId);
   }
 
   @Override
   public String toString() {
-    return "{" +
-      "id='" + id + '\'' +
-      "type='" + type + '\'' +
+    return "MetadataSearchResultRecord{" +
```

Comment:
Is the standard toString for CDAP? I always try to return valid JSON String to make it easier to caller.